### PR TITLE
feat: add tab for submissions view to flowSettings

### PR DESCRIPTION
**Pull request by Mike-Heneghan**
## What

- Add a new placeholder component for a "Feature in development"
- Conditionally render this component as a flow settings tab based on the `SUBMISSION_VIEW` feature flag. 

## Why

- Adding development behind a feature flat allows the work to be shipped incrementally without being visible to user
- Setups the structure for adding a table for visualising application data

## Screenshots

Tab rendered with feature flag enabled:

Screenshot 2024-03-13 at 11 46 18

### CODE REVIEW

File: editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import React from "react";
+import { FeaturePlaceholder } from "ui/editor/FeaturePlaceholder";
+
+const Submissions: React.FC = () => {
+  return (
+    <>
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx:8]: nit: It looks like we have an additional level of fragments here which we don't need

**Review by DafyddLlyr — APPROVED** [review]
One crazy small nitpick sorry, but all good otherwise. Appreciate the small and simple PRs thanks 👍

File: editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx (hunk 1 above)
Comment (Mike-Heneghan) [reply editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx:8]: That's interesting! I directly copy and pasted from an existing tab component and just updated the copy 😅 

I can test it out although I can't think of any reason why we'd need the fragment 🤷

Comment (Mike-Heneghan) [reply editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx]: It definitely seems like the fragment wrapper was unnecessary so I've removed it and from the associated components here: d67cec818356f245ffa8be7e3bc4f39a09c655cf

Thanks for spotting this @DafyddLlyr, I should have been more observant and nice to add a wee tidy up as part of doing this work 😊

Comment (DafyddLlyr) [reply editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx]: Nice outcome to tidy up the other components also 👍

**Comment by Mike-Heneghan** [comment]
[Allow service owners and product owners to see that an application has been sent successfully](https://trello.com/c/Xf8bi4oN/2444-allow-service-owners-and-product-owners-to-see-that-an-application-has-been-sent-successfully)

**Pull request merged**